playwright/tests/page-objects/variables-editor.ts

```diff
@@ -12,8 +12,7 @@ export class VariablesEditor extends Editor {
   }
 
   override async isViewVisible() {
-    const header = this.viewFrameLoactor().locator('.master-toolbar > div > div');
-    expect(header).toHaveText('Variables Editor');
+    const header = this.viewFrameLoactor().locator('.master-toolbar:has-text("Variables Editor")');
```

Comment:
fyi @ivy-lgi: the locator '.master-toolbar > div > div' lead to an error where the selector matched to two results (which cause an error in playwright). And an expect should always have an `await` as it is asynchron.